StudioCore/ParamEditor/ParamBank.cs

```diff
@@ -780,8 +780,14 @@ public static void ReloadParams(ProjectSettings settings, NewProjectOptions opti
                 {
                     if (options.loadDefaultNames)
                     {
-                        new Editor.ActionManager().ExecuteAction(PrimaryBank.LoadParamDefaultNames());
-                        PrimaryBank.SaveParams(settings.UseLooseParams);
+                        try
+                        {
+                            new Editor.ActionManager().ExecuteAction(PrimaryBank.LoadParamDefaultNames());
+                            PrimaryBank.SaveParams(settings.UseLooseParams);
+                        }
+                        catch
```

Comment:
Btw, what's happening here? Suppressing the error feels like a bandaid.

Reply:
Good question. Thanks for the notice